packages/indicators/src/__tests__/__integration/integration.test.ts

```diff
@@ -24,7 +24,7 @@ interface TestCase {
   throws?: boolean;
 }
 
-const models = getTestModels();
+const models = (getTestModels() as unknown) as ModelRegistry;
```

Comment:
eslint-typescript forces this awkward casting as unknown when it can't easily coerce the types. I think until we switch the `@tupaia/database` package to typescript we'll have to stick with something like this...